frontend/database/00207_add_parent_verification.sql

```diff
@@ -0,0 +1,6 @@
+-- Users table
+ALTER TABLE `Users`
+ADD COLUMN `parent_verified` boolean 
+ADD COLUMN `creation_timestamp` timestamp DEFAULT NULL
+ADD COLUMN `random_token_generated` VARCHAR(100) DEFAULT NULL
```

Comment:
Adding the comment:
```suggestion
ADD COLUMN `random_token_generated` VARCHAR(100) DEFAULT NULL COMMENT 'Token que se generará para los usuarios menores de 13 años al momento de registrar su cuenta, el cuál será enviado por correo electrónico al padre',
```
How many characters will have the token? You could use the exact number here